src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    this.color = color;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths must be more that zero');
+    }
+
+    const sortedSides = [a, b, c].sort((x, y) => x - y);
+
+    if (sortedSides[2] >= sortedSides[0] + sortedSides[1]) {
+      throw new Error(`Sides ${a}, ${b}, and ${c} can't form a triangle`);
+    }
+
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  shape = 'circle';
+
+  color: string;
 
-export class Triangle implements Figure {}
+  radius: number;
 
-export class Circle implements Figure {}
+  constructor(color: string, radius: number) {
+    this.color = color;
+
+    if (radius <= 0) {
+      throw new Error('Radius must be greater than zero');
```

Comment:
[CODE STYLE] - The error message could be more descriptive. Instead of 'Radius must be greater than zero', consider specifying that the radius of a circle must be a positive number.